src/xcode/ENA/ENA/Resources/Localization/bg.lproj/Localizable.strings

```diff
@@ -2451,23 +2451,11 @@
 
 "VaccinationCertificate_Registration_description" = "Добавете ваксинационните си сертификати в приложението, за да са винаги с вас. За целта сканирайте QR кода от вашия документ.";
 
-/* - Vaccination Certificate */
-"VaccinationCertificate_title" = "Електронно **Доказателство за ваксиниране**";
-
-"VaccinationCertificate_partiallyVaccinated" = "Ваксинирането не е пълно";
-
-"VaccinationCertificate_vaccinationValidUntil" = "Валидно до %@";
-
```

Comment:
I thought when removing keys in the German file, the keys will be deleted in the other languages files when ever the translation is re-done again. So we do not have to keep care of other files then the German one. Or do I am false?

Reply:
I made a rollback - thank for the notice